{{cookiecutter.project_name}}/.flake8

```diff
@@ -10,32 +10,28 @@ ignore =
     E501
     # whitespace before : -> black does not adhere to PEP8
     E203
+    # line break before binary operator -> black does not adhere to PEP8
+    W503
     # missing whitespace after ,', ';', or ':' -> black does not adhere to PEP8
     E231
     # continuation line over-indented for hanging indent -> black does not adhere to PEP8
     E126
-    # E266 too many leading '#' for block comment -> this is fine for indicating sections
+    # whitespace before ':' -> black does not adhere to PEP8
+    E203
+    # too many leading '#' for block comment -> this is fine for indicating sections
     E262
     # Do not assign a lambda expression, use a def -> lambda expression assignments are convenient
     E731
-    # allow I, O, l as variable names -> I is the identity matrix, i, j, k, l is reasonable indexing notation
+    # allow I, O, l as variable names -> I is the identity matrix
     E741
     # Missing docstring in public package
     D104
-    # ... imported but unused
-    F401
```

Comment:
isort takes care of this... any chance this is required to make this compatible? E.g. if flake8 runs before isort in pre-commit.

Reply:
autoflake does. But flake8 runs after it, so there shouldn't be any issue.